v0/destinations/zendesk/transform.js

```diff
@@ -240,6 +227,48 @@ function getIdentifyPayload(message, category, destinationConfig, type) {
 
   return payload;
 }
+/**
+ * ref: https://developer.zendesk.com/api-reference/ticketing/users/users/#search-users
+ * @param {*} message message
+ * @param {*} headers headers for authorizations
+ * @returns
+ */
+const getUserIdByExternalId = async (message, headers) => {
+  const traits = getFieldValueFromMessage(message, "traits");
+  const externalId =
+    get(traits, "userId") || get(traits, "id") || message.userId;
```

Comment:
```suggestion
const externalId = getFieldValueFromMessage(message, "userIdOnly");
```
Will this not work?

Reply:
Yes, this can work, but I'm not suggesting changing as in this [line](https://github.com/rudderlabs/rudder-transformer/blob/3c41f847d1a18fefcc00f867e054842dad2ff790/v0/destinations/zendesk/transform.js#L206) we are using it in a different order.

Reply:
We can update.

Reply:
done